scripts/globals/wsquest.lua

```diff
@@ -0,0 +1,634 @@
+require('scripts/globals/keyitems');
+require('scripts/globals/quests');
+require('scripts/globals/status');
+
+dsp = dsp or {};
+dsp.wsquest =
+{
+    --------------------------------------
+    -- Quest: The Walls of Your Mind
+    --   NPC: Oggbi (Port Bastok)
+    --    WS: Asuran Fists (Hand-to-Hand)
+    --------------------------------------
+    ['theWallsOfYourMind'] =
+    {
+        ['questName'] = 'The Walls Of Your Mind',
+        ['wsName'] = 'Asuran Fists',
+        ['wsId'] = 9,
+        ['wsUnlockId'] = 1,
+        ['skillId'] = dsp.skill.HAND_TO_HAND,
+        ['minSkill'] = 250,
+        ['trialWeaponName'] = 'Knuckles of Trials',
```

Comment:
Same thing, just describing something already in comment block.

Reply:
I debated removing it as well.